### PR TITLE
[doc] PR for Add separate page for edit kubernetes overrides under manage deployments

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6856

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Generic</a>
 
 ## Prerequisites
 
-Before you start creating a universe, ensure that you performed steps applicable to the cloud provider of your choice, as described in [Configure a cloud provider](/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/).
+Before you start creating a universe, ensure that you performed steps applicable to the cloud provider of your choice, as described in [Configure a cloud provider](../../configure-yugabyte-platform/set-up-cloud-provider/).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone.md:35]: link redirects

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 2]
```diff
@@ -17,8 +17,9 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
...
 
-YugabyteDB Anywhere performs these modifications through the YB-Masters powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+Expansion of universes created with an on-premise cloud provider and secured with third-party certificates obtained from external certification authorities follows a different workflow. For details, see [Expand the universe](../../security/enable-encryption-in-transit#expand-the-universe).
+
+If your universe was created using Kubernetes, you have an option of modifying the Helm chart overrides. To do this, navigate to your universe's **Overiew**, click **Actions > Edit Kubernetes Overrides** to open the **Kubernetes Overrides** dialog and complete it by following instruction provided in [Configure Helm overrides](../../create-deployments/create-universe-multi-zone-kubernetes#configure-helm-overrides).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:24]: by following the instructions provided in

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone.md:35]: fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:24]: fixed

**Comment by lizayugabyte** [comment]
@iSignal Could you take a look at this doc PR, please?

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md (hunk 2 above)
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:24]: The placement of this blurb is confusing. 

Edit universe here refers to the action "Edit Universe" that appears in the UI of a universe. This is different from Edit universe overrides which is a separate action for kubernetes univs. It is probably unlikely the user will discover this blurb here. Can we move it to its own "page" under "manage deployments"?

Also, there was a screenshot attached in the last PR, can we add this screenshot here?

**Review by iSignal — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md]: Changed

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Edit Kubernetes overrides
+headerTitle: Edit Kubernetes overrides
+linkTitle: Edit Kubernetes overrides
+description: Edit Kubernetes overrides for a universe.
+aliases:
+  - /preview/manage/enterprise-edition/edit-helm-overrides/
+menu:
+  preview_yugabyte-platform:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:9]: Need to fix menu this duplicates the old page

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    parent: manage-deployments
+    weight: 60
+type: docs
+---
+
+If your universe was created using Kubernetes, you have an option of modifying the Helm chart overrides. To do this, navigate to your universe's **Overview**, click **Actions > Edit Kubernetes Overrides** to open the **Kubernetes Overrides** dialog shown in the following illustration:
+
+![img](/images/yb-platform/kubernetes-config66.png)
+
+Complete the dialog it by following instructions provided in [Configure Helm overrides](../../create-deployments/create-universe-multi-zone-kubernetes#configure-helm-overrides).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:19]: Complete the dialog by following

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:8]: I don't know what you mean. The menu looks fine in the build preview

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:19]: fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Use YugabyteDB Anywhere to edit a universe
-headerTitle: Edit a universe
-linkTitle: Edit a universe
-description: Use YugabyteDB Anywhere to edit a universe.
+title: Use YugabyteDB Anywhere to modify a universe
+headerTitle: Modify universe
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:3]: changed

**Comment by lizayugabyte** [comment]
@iSignal Could you take a quick look at the changes, please?

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 6]
```diff
@@ -86,13 +92,15 @@ Optionally, complete the **G-Flags** section as follows:
...
   For details, see the following:
 
-  - [Edit configuration flags](/preview/yugabyte-platform/manage-deployments/edit-config-flags)
+  - [Edit configuration flags](../../manage-deployments/edit-config-flags)
 
-  - [YB Master Configuration Flags](/preview/reference/configuration/yb-master/#configuration-flags)
+  - [YB Master configuration flags](../../../reference/configuration/yb-master/#configuration-flags)
 
-  - [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags)
+  - [YB T-Server configuration flags](../../../reference/configuration/yb-tserver/#configuration-flags)
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:99]:
```suggestion
  - [YB-TServer configuration flags](../../../reference/configuration/yb-tserver/#configuration-flags)
```

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 7]
```diff
@@ -72,12 +74,16 @@ Complete the **Instance Configuration** section as follows:
...
 
+### Perform advanced configurations
+
 Complete the **Advanced** section as follows:
 
 - In the **DB Version** field, specify the YugabyteDB version. The default is either the same as the YugabyteDB Anywhere version or the latest YugabyteDB version available for YugabyteDB Anywhere.
 - Use the **Enable IPV6** field to specify whether or not you want to use IPV6 networking for connections between database servers. This setting is disabled by default.
 - Use the **Enable Public Network Access** field to specify whether or not to assign a load balancer or nodeport for connecting to the database endpoints over the internet. This setting is disabled by default.
 
+### Configure g-flags
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:85]:
```suggestion
### Configure G-Flags
```

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 8]
```diff
@@ -151,4 +159,5 @@ You can create a connection to a node as follows:
 
 ## Connect to the universe
 
-For information on how to connect to the universe from the Kubernetes cluster, as well as remotely, see [Connecting YugabyteDB clusters](/preview/deploy/kubernetes/clients/#connecting-tls-secured-yugabytedb-cluster-deployed-by-helm-charts).
+For information on how to connect to the universe from the Kubernetes cluster, as well as remotely, see [Connecting YugabyteDB clusters](../../../deploy/kubernetes/clients/#connecting-tls-secured-yugabytedb-cluster-deployed-by-helm-charts).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:162]:
```suggestion
For information on how to connect to the universe from the Kubernetes cluster, as well as remotely, see [Connecting YugabyteDB clusters](../../../deploy/kubernetes/clients/#connect-tls-secured-yugabytedb-cluster-deployed-by-helm-charts).
```

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+title: Edit Kubernetes overrides
+headerTitle: Edit Kubernetes overrides
+linkTitle: Edit Kubernetes overrides
+description: Edit Kubernetes overrides for a universe.
+aliases:
+menu:
+  preview_yugabyte-platform:
+    identifier: edit-helm-overrides
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:9]: Any reason why the file name and the identifier is `edit-helm-overrides` and not `edit-kubernetes-overrides` ?

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:99]: fixed

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:85]: fixed

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:162]: fixed

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md (hunk 9 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md:9]: helm is shorter than kubernetes :)

**Comment by lizayugabyte** [comment]
This PR has been open for a while now and the files can accumulate potential merge conflicts. I'm going to merge the PR now. Any additional requests for changes can be addressed via a new PR.

**Pull request merged**